src/stash/src/postgres.rs

```diff
@@ -528,6 +528,11 @@ impl Stash {
                 tracing::error!("postgres stash connection error: {}", e);
             }
         });
+        // The Config is shared with the Consolidator, so we update the application name in the
```

Comment:
It's not entirely clear to me why this is, maybe so that they can share a connection timeout? I'm not sure that it's worth the cost of a mutex though.